README.md

```diff
@@ -21,10 +21,14 @@ To run the app in a browser with frontend and contract hot-reloading, simply run
 ### npm Scripts
 
 - **postinstall**: Runs after installing dependencies.
+- **copy-artifacts**: Add aragonOS artifacts pre-compiled.
```

Comment:
We should have a better explanation for why this is (currently) needed, and how it's used.

I might not even put it here, but maybe under an "Additional information" section or etc.